test-status/api.js

```diff
@@ -129,6 +129,40 @@ function createReportedCheckParams(
   return params;
 }
 
+/**
+ * Create a parameters object for reporting an errored test to an existing
+ * GitHub status check.
+ *
+ * @param {string} pullRequestSnapshot pull request snapshot from database.
+ * @param {string} type tests type slug.
+ * @param {number} checkRunId the existing check run ID.
+ * @return {!Object} a parameters object for github.checks.update.
+ */
+function createErroredCheckParams(pullRequestSnapshot, type, checkRunId) {
+  const {owner, repo, head_sha: headSha} = pullRequestSnapshot;
+  const detailsUrl = new URL(
+      `/tests/${headSha}/${type}/status`, process.env.WEB_UI_BASE_URL);
+  return {
+    owner,
+    repo,
+    check_run_id: checkRunId,
+    status: 'completed',
+    completed_at: new Date().toISOString(),
+    details_url: detailsUrl.href,
+    conclusion: 'action_required',
+    output: {
+      title: `${type} tests have errored`,
+      summary: `An unexpected error occurred while running ${type} tests.`,
+      text: 'Please inspect the Travis build for the details.\n\n' +
+        'If you believe that this pull request was not the cause of this ' +
+        'test error (i.e., this is a flaky test) please contact the weekly ' +
```

Comment:
nit: Restate to `... was not the cause of this error, please contact...`

Reply:
Done